apps/web/app/(all)/[workspaceSlug]/(projects)/analytics/[tabId]/page.tsx

```diff
@@ -82,18 +83,20 @@ const AnalyticsPage = observer((props: Props) => {
         <>
           {workspaceProjectIds.length > 0 || loader === "init-loader" ? (
             <div className="flex h-full overflow-hidden bg-custom-background-100 justify-between items-center  ">
-              <Tabs
-                tabs={tabs}
-                storageKey={`analytics-page-${currentWorkspace?.id}`}
-                defaultTab={defaultTab}
-                size="md"
-                tabListContainerClassName="px-6 py-2 border-b border-custom-border-200 flex items-center justify-between"
-                tabListClassName="my-2 w-auto"
-                tabClassName="px-3"
-                tabPanelClassName="h-full overflow-hidden overflow-y-auto px-2"
-                storeInLocalStorage={false}
-                actions={<AnalyticsFilterActions />}
-              />
+              <AnalyticsWorkspaceProvider workspaceSlug={params.workspaceSlug}>
```

Comment:
@prateekshourya29 this was an attempt to move away from params and create some abstraction for data